p2p/peer.py

```diff
@@ -577,16 +606,30 @@ def msg_queue(self) -> 'asyncio.Queue[PEER_MSG_TYPE]':
     def queue_size(self) -> int:
         return self.msg_queue.qsize()
 
-    def add_msg(self, msg: 'PEER_MSG_TYPE') -> None:
+    def add_msg(self, msg: 'PEER_MSG_TYPE') -> bool:
         peer, cmd, _ = msg
+
+        if not self.is_subscription_command(type(cmd)):
+            if hasattr(self, 'logger'):
```

Comment:
Which subscribers don't provide a logger? Should we change them so they do?

Reply:
This is actually for the tests.  The `PeerSubscriber` itself doesn't provide a logger.  I'll remove this and change the ones in the test to have loggers.